### PR TITLE
docs: TTL policy

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

as title

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/manage-data/overview.md [hunk 1]
```diff
@@ -277,4 +277,35 @@ TRUNCATE TABLE monitor;
...
+) WITH ('ttl'='7d');
+```
+
+你还可以创建数据库级别的 TTL 策略。
+例如，以下 SQL 语句创建了一个名为 `test` 的数据库，并设置了 7 天的 TTL 策略：
+
+```sql
+CREATE DATABASE test WITH ('ttl'='7d');
+```
+
```
Comment (killme2008) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/manage-data/overview.md:308]: 可能需要加一些说明，设置了数据库的 ttl，还是可以继续设置表的  ttl，但是如果表没有明确设置，将在创建的时候继承数据库的设置。

**Comment by nicecui** [comment]
@killme2008 PTAL

**Review by fengjiachun — APPROVED** [review]
LGTM

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**